### DISCUSSION TITLE
.pdb file not applying in stack traces

### DISCUSSION THREAD

**Discussion by pierrelasse** [YimMenu/YimMenu#3005] [Q&A] [answered]
I have it like this
![image](https://github.com/YimMenu/YimMenu/assets/76223390/807b6fe6-0d3a-483f-8c56-4911f4c9870c)
And I'm injecting using Xenos.
Do I need to do anything to "apply" the .pdb file so stacktraces are formatted with a file path?
Because it's currently just giving me a `YimMenu.dll+0xsomething`.

**Comment by Yimura** [comment, marked as answer]
Did you build these or are these from a build artifact on Github? Otherwise you need to patch the PDB path in the YimMenu dll.

**Reply by pierrelasse** [reply]
Nop i got it from github actions. I also just noticed that I have to change the patch in the exe.

**Reply by xiaoxiao921** [reply]
you can also do it through winapi calls. down the line this could be handled by env variable

SymInitialize UserSearchPath / SymSetSearchPath
https://learn.microsoft.com/en-us/windows/win32/api/dbghelp/nf-dbghelp-syminitialize

**Reply by pierrelasse** [reply]
This is the path used in the dll by default `C:/action-runner/_work/YimMenu/YimMenu/build/YimMenu.pdb`.

Edit: Would a relative path work? In theory.

**Reply by xiaoxiao921** [reply]
SymSetSearchPath make it so that the pdb link embedded in the .exe is basically irrelevant.
